### PR TITLE
Update contribution guide and learning guide

### PR DESCRIPTION
**Pull request by Olshansk**
## Description
Update the learning and contribution guides in v1.

## Type of change
Please mark the options that are relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation
- [ ] Other (<_REPLACE_ME_WITH_DETAILS_>)

## How Has This Been Tested?
NA

### CODE REVIEW

**Comment by Olshansk** [comment]
[@deblasis](https://github.com/deblasis) - We talked about an easy way to learn more about Pocket: check out `docs/learning/README.md` :)

**Comment by Olshansk** [comment]
[@b3nnnp](https://github.com/b3nnnp) - Check out `docs/contributing/README.md`

File: docs/contributing/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# How to contribute?
+
+We have an [extensive contribution guide](docs/contributing/CONTRIBUTING.md) outlining all the steps to contribute, but if you just want to start getting involved, follow the next two steps:
+
+1. Look at available work on [app.dework.xyz/pokt-network](https://app.dework.xyz/pokt-network) or look at our [starter tasks](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22starter+task%22) / [community](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22community%22+) tasks on Github.
+
+2. Reach out to the team on Discord in the [#v1-dev channel](https://discord.com/channels/553741558869131266/986789914379186226) or [#v1-research channel](https://discord.com/channels/553741558869131266/936666517498650644).
```
Comment (deblasis) [inline docs/contributing/README.md:7]: Not sure if by design but #v1-dev on discord is read-only, at least for my user 🙂

**Review by phthan0 — APPROVED** [review]
Acceptable

**Comment by Olshansk** [comment]
@phthan0 Where on the acceptable scale does this lie?

File: docs/contributing/README.md (hunk 1 above)
Comment (Olshansk) [reply docs/contributing/README.md]: Lol, it is.
Screen Shot 2022-08-05 at 10 17 56 PM

Try again :)

**Comment by phthan0** [comment]
4

**Pull request merged**